### ISSUE TITLE
[Usage]: Getting logits of the first token to generate of a multi-modal model

### ISSUE THREAD

**Issue by marcmk6** [vllm-project/vllm#20171]
### Your current environment

v1 engine

### How would you like to use vllm

I want the logits of the first-token-to-generate over the vocabulary. I can successfully get a hidden embedding. Am I supposed to multiply it with the `lm_head`?
The vocab distribution I got in this way seems quite different from the logprobs I get from `generate`.

### Before submitting a new issue...

- [x] Make sure you already searched for relevant issues, and asked the chatbot living at the bottom right corner of the [documentation page](https://docs.vllm.ai/en/latest/), which can answer lots of frequently asked questions.

**Comment by marcmk6** [comment]
Figured out: the default pooling config normalizes the embedding. 
```python
pooler_config = PoolerConfig(
      pooling_type="LAST",
      #   normalize=True,
      normalize=False, # don't normalize
)
```
